macros/generic_tests/fewer_rows_than.sql

```diff
@@ -13,15 +13,21 @@
       and a.{{c}} = b.{{c}}
     {% endfor %}
   {% endset %}
+  {% set groupby_gb_cols = 'group by ' + group_by_columns|join(',') %}
 {% endif %}
-{% set group_by_columns = ['id'] + group_by_columns %}
+
+{#-- We must add a fake join key in case additional grouping variables are not provided --#}
+{#-- Redshift does not allow for dynamically created join conditions (e.g. full join on 1 = 1 --#}
+{#-- The same logic is used in fewer_rows_than. In case of changes, maintain consistent logic --#}
```

Comment:
Guessing this is a copy-paste mishap?

```suggestion
{#-- The same logic is used in equal_rowcount. In case of changes, maintain consistent logic --#}
```

Reply:
😳 Yes indeed! Thanks for the catch